Source/Core/Core/PowerPC/GDBStub.cpp

```diff
@@ -309,10 +309,18 @@ static void HandleQuery()
 {
   DEBUG_LOG_FMT(GDB_STUB, "gdb: query '{}'", CommandBufferAsString() + 1);
 
-  if (!strcmp((const char*)(s_cmd_bfr + 1), "TStatus"))
-  {
-    return SendReply("T0");
-  }
+  if (!strcmp((const char*)(s_cmd_bfr + 1), "Attached"))
+    return SendReply("1");
```

Comment:
The [documentation](https://sourceware.org/gdb/onlinedocs/gdb/General-Query-Packets.html#index-qAttached-packet) says that the syntax is <code>qAttached:<var>pid</var></code>.  To my understanding, `strcmp` returns a nonzero value if the strings are of different lengths (i.e. `strcmp("qAttached:1", "qAttached")` is nonzero).  So I'm not sure if this will do anything.  You seem to use `strncmp` for this below; perhaps it should be used here?

Reply:
So the pid seems to be optional because it sends `qAttached` on my end, but since we are never going to create a new process on attach, I might as well do it sure.

Reply:
Ah, the documentation does explicitly say that:

> When the multiprocess protocol extensions are supported (see multiprocess extensions), <var>pid</var> is an integer in hexadecimal format identifying the target process. Otherwise, GDB will omit the <var>pid</var> field and the query packet will be simplified as ‘`qAttached`’.

I think it is appropriate to respond with an error if a PID is specified, then.

Reply:
Well, we never reply with multiprocess+ in qSupported, I don't think adding the if for that is worth when if that even happens, the problem would be in qSupported.